### DISCUSSION TITLE
How can I swap my github email?

### DISCUSSION THREAD

**Discussion by BlueFalconHD** [community/community#39484] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I am switching my Github account email from gMail to iCloud due to security issues and ease of access.
My current email settings look like: (email addresses blurred for privacy. They are the same one)
<img width="808" alt="image" src="https://user-images.githubusercontent.com/72631767/202826812-32eee56a-f9fd-4b5a-86b0-1289f339bc26.png">
When I try to add my iCloud address, this error appears.
<img width="1412" alt="image" src="https://user-images.githubusercontent.com/72631767/202826940-cd8c6006-8512-4ebd-b274-169cca8ef7be.png">
As you can see I am not using that address on my account, yet it says I am. I am not exactly sure what is happening here. The error message showing isn't the most clear. I am not sure if it means another account is using that email, but I have never made an account with that address.

**Comment by jfullstackdev** [comment]
yup, that can really be an issue, maybe Support can help identifying that problem

that's why I'm adding all my emails even though some are inactive, just to not be taken

https://support.github.com/contact/account

maybe some bot is using it ? or the email cannot really be used ? not sure

**Comment by odysseuskir** [comment]
Here's what I would do: 

1. Open a private window  
2. Go to GitHub 
3. Login using the iCloud email 
4. Reset the password 
5. Login with a new password 
6. Go to settings 
7. Delete the account 
8. Go back to your main account
9. Try adding the iCloud email 
10. Set it as primary

If something goes wrong, maybe you should contact support [here](https://support.github.com/contact).

Good luck!

**Comment by BlueFalconHD** [comment, marked as answer]
Ok! I found out what happened. I have a very old account registered with that email. After deleting that account, this issue was resolved! Thanks for all the help guys!
